settings.gradle.kts

```diff
@@ -50,8 +50,8 @@ module(name = ":benchmark", path = "benchmark")
 // Version Catalog
 dependencyResolutionManagement {
     versionCatalogs {
-        create("libs") {
-            from(files("libs.versions.toml"))
+        create("exampleLibs") {
+            from(files("gradle/example.libs.versions.toml"))
```

Comment:
This idea is really good!

It's nice to have a separate list of libraries on the product side for the minimum version to ensure compatibility and on the example side to make sure they work on the latest version. 

I will update the libraries used in the example and the product will be fixed after this PR is merged.

Reply:
Is it possible to configure toml in the top-level instead of in the gradle folder? Because I want to show people who come to see what versions of the library the Kotlin JDSL is using at the top-level.